suites/compaction/rgw/rgw.yaml

```diff
@@ -0,0 +1,131 @@
+tests:
+
+  # Tests from - suites/squid/rgw/tier-1_rgw.yaml
+  - test:
+      abort-on-fail: true
+      config:
+        haproxy_clients:
+          - node4
+        rgw_endpoints:
+          - node2:80
+          - node3:80
+      desc: "Configure HAproxy"
+      module: haproxy.py
+      name: "Configure HAproxy"
+
+  - test:
+      name: enable bucket versioning
+      desc: Basic versioning test, also called as test to enable bucket versioning
+      polarion-id: CEPH-14261  # also applies to CEPH-9222 and CEPH-10652
+      module: sanity_rgw.py
+      config:
+        script-name: test_versioning_with_objects.py
+        config-file-name: test_versioning_objects_enable.yaml
+
+  - test:
+      name: Test M buckets with N objects
+      desc: test to create "M" no of buckets and "N" no of objects
+      polarion-id: CEPH-9789
+      module: sanity_rgw.py
+      config:
+        script-name: test_Mbuckets_with_Nobjects.py
+        config-file-name: test_Mbuckets_with_Nobjects.yaml
+        install_common: false
+        run-on-rgw: true
+
+  - test:
+      name: Test delete using M buckets with N objects
+      desc: test to create "M" no of buckets and "N" no of objects with delete
+      polarion-id: CEPH-14237
+      module: sanity_rgw.py
+      config:
+        script-name: test_Mbuckets_with_Nobjects.py
+        config-file-name: test_Mbuckets_with_Nobjects_delete.yaml
+        install_common: false
+        run-on-rgw: true
+
+  - test:
+      name: Test download with M buckets with N objects
+      desc: test to create "M" no of buckets and "N" no of objects with download
+      polarion-id: CEPH-14237
+      module: sanity_rgw.py
+      config:
+        script-name: test_Mbuckets_with_Nobjects.py
+        config-file-name: test_Mbuckets_with_Nobjects_download.yaml
+        install_common: false
+        run-on-haproxy: true
+
+  - test:
+      name: Test multipart upload of M buckets with N objects
+      desc: test to create "M" no of buckets and "N" no of objects with multipart upload
+      polarion-id: CEPH-9801
+      module: sanity_rgw.py
+      config:
+        script-name: test_Mbuckets_with_Nobjects.py
+        config-file-name: test_Mbuckets_with_Nobjects_multipart.yaml
+        install_common: false
+        run-on-rgw: true
+
+  - test:
+      name: test_bi_purge for a bucket
+      desc: test bi_purge should not error
+      polarion-id: CEPH-83575234
+      module: sanity_rgw.py
+      config:
+        install_common: false
```

Comment:
please remove install_common: false form config
We need install_common only for parallel run

Reply:
done